plat/linuxu/Config.uk

```diff
@@ -1,7 +1,7 @@
 menuconfig PLAT_LINUXU
        bool "Linux user space"
        default n
-       depends on ((ARCH_X86_64 && !HAVE_SYSCALL && !HAVE_SMP) || (ARCH_ARM_32 && !HAVE_SYSCALL && !HAVE_SMP))
+       depends on (!HAVE_SYSCALL && !HAVE_SMP)
```

Comment:
Hey, I noticed that we can't compile arm64 linux with scheduling at the moment. So, you should use the following:
```
depends on !HAVE_SYSCALL && !HAVE_SMP && ((!ARCH_ARM_32 && !ARCH_ARM_64) || !HAVE_SCHED)
```